src/dos/dos_keyboard_layout.cpp

```diff
@@ -331,7 +331,11 @@ Bitu keyboard_layout::read_keyboard_file(const char* keyboard_file_name, Bit32s
 			return KEYB_FILENOTFOUND;
 		}
 		if (tempfile) {
-			fseek(tempfile.get(), start_pos + 2, SEEK_SET);
+			if (fseek(tempfile.get(), start_pos + 2, SEEK_SET) != 0) {
+				LOG(LOG_BIOS, LOG_ERROR)
+				("Keyboard layout file %s not found", keyboard_file_name);
+				return KEYB_INVALIDFILE;
+			}
```

Comment:
At this point, it's more likely that some other error occurred, and this error will just mask the true error.

The proper thing to do would be to check for `errno`, and log it.

Reply:
Thanks @shermp.
The LOG_MSG now also includes the `errno` string.